Kconfig.zephyr

```diff
@@ -575,6 +575,7 @@ endif # BUILD_OUTPUT_UF2
 
 config BUILD_OUTPUT_STRIPPED
 	bool "Build a stripped binary"
+	default y
```

Comment:
As per other commit, rejecting as this is not something that should ever be enabled by default